tests/integrations/docker/test_DockerAppContext__verify.py

```diff
@@ -100,7 +100,7 @@ def test_docker_verify_fail(mock_tools, first_app_config, verify_kwargs, monkeyp
 def test_docker_image_build_fail(mock_tools, first_app_config, verify_kwargs):
     """Failure if Docker image build fails."""
     mock_tools.subprocess._subprocess.Popen.side_effect = subprocess.CalledProcessError(
-        returncode=80, cmd=["docker" "build"]
+        returncode=80, cmd=["dockerbuild"]
```

Comment:
This should be 2 strings, not one concatenated string
```suggestion
        returncode=80, cmd=["docker", "build"]
```
I'm guessing it was split on 2 lines at some point, but the comma was omitted, and black autocleaned it and we didn't notice.